pandas/_libs/tslibs/offsets.pyx

```diff
@@ -756,18 +756,27 @@ cdef class BaseOffset:
         raise ValueError(f"{self} is a non-fixed frequency")
 
     def is_anchored(self) -> bool:
-        # TODO: Does this make sense for the general case?  It would help
-        # if there were a canonical docstring for what is_anchored means.
+        # GH#55388
         """
         Return boolean whether the frequency is a unit frequency (n=1).
 
+        .. deprecated:: 2.2.0
+            The is_anchored is deprecated and will be removed in a future version.
```

Comment:
"The is_anchored is deprecated" -> "is_anchored is deprecated"

Reply:
it's done